src/async_wrap.cc

```diff
@@ -162,19 +162,25 @@ static void DestroyAsyncIdsCallback(void* arg) {
 }
 
 
-void AsyncWrap::EmitPromiseResolve(Environment* env, double async_id) {
+void Emit(Environment* env, double async_id, AsyncHooks::Fields type,
```

Comment:
perhaps `inline void Emit(...`?

Reply:
@jasnell Imo we shouldn’t be adding `inline` to functions that aren’t defined in the corresponding `-inl.h` header

Also, @bnoordhuis pointed out somewhere else that we should put the `inline` specifier on the declarations in the `.h` header file

Reply:
ok, I'm good with that.